From: Ian Jackson <ian.jackson@citrix.com>
Date: Fri, 12 Oct 2018 17:56:56 +0100
Subject: docs/man/xen-vbd-interface.7: Provide properly-formatted NAME
 section

This manpage was omitted from
   docs/man: Provide properly-formatted NAME sections
because I was previously building with markdown not installed.

Signed-off-by: Ian Jackson <ian.jackson@citrix.com>
---
 docs/man/xen-vbd-interface.7.pandoc | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/docs/man/xen-vbd-interface.7.pandoc b/docs/man/xen-vbd-interface.7.pandoc
index ba0d159..2f18d5b 100644
--- a/docs/man/xen-vbd-interface.7.pandoc
+++ b/docs/man/xen-vbd-interface.7.pandoc
@@ -1,3 +1,8 @@
+Name
+----
+
+xen-vbd-interface - Xen paravirtualised block device protocol
+
 Xen guest interface
 -------------------
 
